test/e2e/setup/setup.go

```diff
@@ -56,7 +61,7 @@ func Setup(t *testing.T) *e2eutil.Context {
 
 // CreateTLSResources will setup the CA ConfigMap and cert-key Secret necessary for TLS
 // The certificates and keys are stored in testdata/tls
-func CreateTLSResources(namespace string, ctx *e2eutil.Context) error { //nolint
+func CreateTLSResources(namespace string, ctx *e2eutil.Context, secretType tlsSecretType) error { //nolint
```

Comment:
now related to this PR, curious why do we have a `//nolint` here ?

Reply:
good question :) next commit I'll try to remove it and see what happens!

Reply:
golanci-lint does not raise any error, removed!